apps/hash-api/src/integrations/linear/sync-back.ts

```diff
@@ -70,20 +70,11 @@ export const processEntityChange = async (
 
   switch (entityTypeId) {
     case linearTypes.entityType.issue.entityTypeId: {
-      // eslint-disable-next-line no-console
-      console.log("Change to Linear Issue detected");
-
-      // @todo check this works
-      const result = await temporalClient.workflow.start("updateLinearIssue", {
+      await temporalClient.workflow.start("updateLinearIssue", {
```

Comment:
Do we want to wait for completion instead? Then we could check if the sync was successful.

Reply:
I think we should handle that via visibility into Temporal job failures, not returning it to the API – there's no user request to pass the error back to so the most we could do is log an error somewhere